roles/pulp/tasks/main.yml

```diff
@@ -27,21 +36,7 @@
 - name: set the default system PATH as a fact
   set_fact:
     default_bin_path: "{{ systemd_show_env_path.stdout }}"
-
-
-# Try multiple possible names for the rhel7 optional repo until it is found.
-# The query ensures that a single string rather than a list of strings is valid.
-- include_tasks: rhel7-optional.yml
-  with_items: "{{ rhel7_optional_repo }}"
-  when:
-    - ansible_distribution == "RedHat"
-    - ansible_distribution_major_version|int == 7
-    # Works for both strings and lists to make sure not empty
-    - rhel7_optional_repo is not none
-    - rhel7_optional_repo | length > 0
-    # Prevents running again once completed.
-    - optional_repo_enabled is not defined
-
+- import_tasks: repos.yml
```

Comment:
cool!

Reply:
@fabricio-aguiar  Yeah, I try to do some cleanup / refactoring whenever I am touching related code.

I'm more proud of converting rhel7-optional.yml to a generic re-usable tasks file now, ambiguously-named-repo.yml